packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -179,12 +179,14 @@ export class LoginPage {
 				await Promise.all( [
 					// Shorter than usual timoout, because with a cookie file the login process
 					// should not take more than a few seconds.
-					this.page.waitForNavigation( { url: landingUrl, waitUntil: 'load', timeout: 15 * 1000 } ),
+					this.page.waitForNavigation( { url: landingUrl, waitUntil: 'load', timeout: 7 * 1000 } ),
 					this.page.goto( getCalypsoURL( '/' ) ),
 				] );
 				return;
-			} catch {
-				console.log( 'Unable to log in using cookie file, retrying a normal login.' );
+			} catch ( err: unknown ) {
+				if ( err instanceof Error ) {
```

Comment:
Do we need to check the instance here? Will it retry the username/password-based authentication regardless of the error type?

Reply:
TypeScript tells me I need to check the `err` type in the catch block, so I've implemented it this way.
https://stackoverflow.com/a/68257472

Reply:
For the second part of your question, yes, the normal username/password attempt will be made regardless of the error type.